iphone/Maps/UI/Search/Filters/MWMSearchHotelsFilterViewController.mm

```diff
@@ -1,8 +1,43 @@
 #import "MWMSearchHotelsFilterViewController.h"
 #import "MWMSearchFilterViewController_Protected.h"
+#import "SwiftBridge.h"
+
+#include "base/stl_helpers.hpp"
+
+#include <array>
+#include <vector>
 
 namespace
 {
+
+std::array<ftypes::IsHotelChecker::Type, static_cast<size_t>(ftypes::IsHotelChecker::Type::Count)> const kTypes = {{
+  ftypes::IsHotelChecker::Type::Hotel,
```

Comment:
Can we sort here, or there is predefined order/priority?
If so, maybe we should add a link to the doc page?